dashboard/test/ui/features/blockTrashing.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Blocks can be trashed in certain circumstances
+
+Background:
+  Given I am on "http://learn.code.org/s/course2/stage/19/puzzle/2"
+  # In this stage:
+  # block 18 is the "when run" block
+  # block 19 is an undeletable "set color" block
+  # block 24 is a deletable "turn right" block
```

Comment:
Is it easy to alias these values in a script, so I don't use the numbers throughout?